src/App.tsx

```diff
@@ -1,7 +1,39 @@
-import React from 'react';
+import React, { Component } from 'react';
 
-export const App: React.FC = () => (
-  <div className="App">
-    <p className="App__message">The last pressed key is [Enter]</p>
-  </div>
-);
+type State = {
+  pressedKey: string;
```

Comment:
let's assign null if nothing pressed yet
```suggestion
  pressedKey: string | null;
```

Reply:
done!